intel-sgx/pcs/src/tcb_evaluation_data_numbers.rs

```diff
@@ -294,5 +299,56 @@ mod tests {
             assert!(false);
         }
     }
+
+    #[test]
+    fn tcb_eval_number() {
+        let april_8_2025 = Utc.with_ymd_and_hms(2025, 4, 8, 14, 55, 0).unwrap();
+        let april_9_2025 = Utc.with_ymd_and_hms(2025, 4, 9, 14, 55, 0).unwrap();
+        let april_15_2025 = Utc.with_ymd_and_hms(2025, 4, 15, 14, 55, 0).unwrap();
+        let april_16_2025 = Utc.with_ymd_and_hms(2025, 4, 16, 14, 55, 0).unwrap();
+        let april_17_2025 = Utc.with_ymd_and_hms(2025, 4, 17, 14, 55, 0).unwrap();
+        let number = TcbEvalNumber {
+            number: 42,
+            tcb_recovery_event_date: april_9_2025,
+            tcb_date: april_9_2025,
+        };
+
+        let policy = TcbPolicy::new(Duration::days(8));
+        assert!(!policy.needs_to_be_enforced(&number, &april_8_2025));
+        assert!(!policy.needs_to_be_enforced(&number, &april_9_2025));
+        assert!(!policy.needs_to_be_enforced(&number, &april_15_2025));
+        assert!(!policy.needs_to_be_enforced(&number, &april_16_2025));
+        assert!(policy.needs_to_be_enforced(&number, &april_17_2025));
+    }
+
+    #[test]
+    fn minimum_tcb_evaluation_data_number() {
+        let numbers = RawTcbEvaluationDataNumbers::read_from_file("./tests/data").unwrap();
+        let numbers: TcbEvaluationDataNumbers<Unverified> = serde_json::from_str(numbers.raw_tcb_evaluation_data_numbers()).unwrap();
+
+        let april_8_2025 = Utc.with_ymd_and_hms(2025, 4, 8, 0, 0, 0).unwrap();
+        let march_12_2024 = Utc.with_ymd_and_hms(2024, 3, 12, 0, 0, 0).unwrap();
+        let march_13_2024 = Utc.with_ymd_and_hms(2024, 3, 13, 0, 0, 0).unwrap();
+        let november_12_2024 = Utc.with_ymd_and_hms(2024, 11, 12, 0, 0, 0).unwrap();
+        let november_13_2024 = Utc.with_ymd_and_hms(2024, 11, 13, 0, 0, 0).unwrap();
+        let november_20_2024 = Utc.with_ymd_and_hms(2024, 11, 20, 0, 0, 0).unwrap();
+        let number_17 = TcbEvalNumber {
+            number: 17,
+            tcb_recovery_event_date: march_12_2024.clone(),
```

Comment:
Do you need the clone() here and in the other dates below? It can be cleaned up.